.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: release
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository_owner == 'flutter-tizen' }}
+    steps:
+    - uses: subosito/flutter-action@v1
+      with:
+        channel: stable
+    - uses: actions/checkout@v2
+      with:
+          fetch-depth: 2
+    - name: Set up tools
+      run: dart pub get
+      working-directory: ${{ github.workspace }}/tools
+    - name: Dry run on pull request
+      if: ${{ github.event_name == 'pull_request' }}
+      run: |
+        ./tools/tools_runner.sh publish-plugin \
+          --all-changed \
+          --pub-publish-flags=--dry-run \
+          --base-sha=$(git rev-parse HEAD~)
+    - name: Wait on all tests
+      if: ${{ github.event_name == 'push' }}
+      uses: lewagon/wait-on-check-action@v1.1.1
+      with:
+        ref: ${{ github.sha }}
+        running-workflow-name: 'release'
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        wait-interval: 180 # seconds
+        allowed-conclusions: success
+        verbose: false
+    - name: Publish plugins
+      if: ${{ github.event_name == 'push' }}
+      run: |
+        ./tools/tools_runner.sh publish-plugin \
+          --all-changed \
+          --pub-publish-flags=--dry-run \
+          --base-sha=HEAD~ \
+        ./tools/tools_runner.sh publish-plugin \
+          --all-changed \
+          --base-sha=HEAD~ \
```

Comment:
What's different from the above `--base-sha=$(git rev-parse HEAD~)`?

Reply:
Oh, they aren't different in terms of program execution. I'll use `HEAD~`, thanks!